tests/integrations/templates/welcome.html

```diff
@@ -8,10 +8,15 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <script src="https://js.pusher.com/7.0/pusher.min.js"></script>
+    <link rel="stylesheet" href="/static/main.css">
 </head>
 <body>
     <h1>Welcome</h1>
 
+    <div>
+        <img src="{{ asset('local', 'original/image-20150902-6700-t2axrz.jpg') }}"
```

Comment:
maybe put a more generic image here ?

Reply:
This is just an example, it will be removed on final commit